Makefile

```diff
@@ -93,6 +105,10 @@ $(BUILD_IMAGE_MARKER): $(BUILD_DIR)/ipmanager
 	echo > $(BUILD_IMAGE_MARKER)
 
 
+$(VENDOR_DIR):
+	$(DOCKER_EXEC) bash -xc 'go get github.com/Masterminds/glide && glide install --strip-vendor'
```

Comment:
Let's also change this part a bit to avoid `root:root` dirs/files:
```
$(VENDOR_DIR):
  $(DOCKER_EXEC) bash -xc 'go get github.com/Masterminds/glide \
    && glide install --strip-vendor; \
    chown $(shell id -u):$(shell id -u) -R vendor'
```